layers/+source-control/version-control/funcs.el

```diff
@@ -59,7 +59,7 @@
   (let ((current-prefix-arg t))
     (call-interactively
      (cl-case version-control-diff-tool
-       (diff-hl     'diff-hl-margin-minor-mode)
+       (diff-hl     'diff-hl-margin-mode)
```

Comment:
> It's not autoloaded, which seems to be on purpose. Any idea why this function is used and not the plain mode?

It has been introduced [here](https://github.com/syl20bnr/spacemacs/pull/3632/files#diff-7b025371ed8168e51c1dce9d7c919981R63).

@ralesi Did you use `diff-hl-margin-mode` in purpose, or would `diff-hl-mode` be fine? (ref. https://github.com/syl20bnr/spacemacs/issues/4034)